cdap-explore-client/src/main/java/co/cask/cdap/explore/client/ExploreHttpClient.java

```diff
@@ -329,6 +347,24 @@ public QueryHandle getTypeInfo() throws ExploreException, SQLException {
     throw new ExploreException("Cannot get the tables. Reason: " + getDetails(response));
   }
 
+  @Override
+  public QueryHandle createNamespace(Id.Namespace namespace) throws ExploreException, SQLException {
+    HttpResponse response = doPut(String.format("data/explore/namespaces/%s", namespace.getId()), null, null);
```

Comment:
namespace here not top level. The API should be 
`namespaces/{namespaceId}/data/explore`